.travis.yml

```diff
@@ -17,3 +17,10 @@ cache:
     - node_modules
     - test/fixtures/node_modules
     - test/fixtures/bower_components
+notifications:
+  webhooks:
+    urls:
+      - secure: "DhPNqHXuUIeIGE9Ek3+63qhco+4MozXqMZL6dAKoq1MHQ2RAPO6SYIkUYZqDnuWYlwWao2EnTYcDREivIV/m/RnkP9bKlpX/n/RNJe+X4bwFaCU55fVKgkAFn3takSBC5SVoeTWHdWu3WhhqSdioWjT7mlE1wtt/RanSMb5Id8M="
```

Comment:
Note, this does prevent PR's from forked repos from pushing activity as well; that may need to be considered.